kolibri/plugins/learn/assets/src/vue/toolbar/index.vue

```diff
@@ -0,0 +1,238 @@
+<template>
+
+  <div v-bind:class="['toolbar-show', displayToolBar ? 'toolbar-hide' : '' ]" >
+    <breadcrumbs class="breadcrumbs"></breadcrumbs>
+    <div :class="{ 'toggle-menu-on' : more }">
+      <label for="chan-select" :class="[ more ? 'lable-on' : 'visuallyhidden' ]" >Switch Channels</label>
+      <select
+        class="chan-select"
+        :class="[ more ? 'chan-select-mobile-location' : 'chan-select-location' ]"
+        id="chan-select"
+        name="chan-select"
+        v-model="getCurrentChannel"
```

Comment:
Unfortunately, this has already been merged, but you should probably avoid `v-model` here, as that does two way data binding.

As the computed property you are using does not have a setter, `v-model` is not appropriate, however, if you wanted, you could add a setter to the computed property, to trigger the action that will update the vuex store.